test/Node/OfflineTest.php

```diff
@@ -58,31 +59,25 @@ public function testCreateFromArrayObjectDn()
         $this->assertFalse($node->isAttached());
     }
 
-    /**
-     * @expectedException Laminas\Ldap\Exception\ExceptionInterface
-     */
     public function testCreateFromArrayMissingDn()
     {
+        $this->expectException(LdapException::class);
```

Comment:
Please move the `->expectException()` calls throughout this patch right before the call that would lead to an exception (requires a general search for `expectException()` usages)

Reply:
Done in PR #13